tron/bin/check_tron_jobs.py

```diff
@@ -52,6 +53,13 @@ def parse_cli():
         dest="run_interval",
         default=300,
     )
+    parser.add_argument(
+        "--skip-failed-logging-for-superregion",
```

Comment:
might be nice to make this match the name of the puppet toggle for this (e.g, `skip-sensu-failure-logging` or something) so that folks don't need to wonder why this option references superregions